### ISSUE TITLE
Grouped icon transformations not applied correctly

### ISSUE THREAD

**Issue by cristianvogel** [fomantic/Fomantic-UI#1740]
# Bug Report
When using grouped icons, and attempting to synthesise a variation by grouping and rotating, seems like the rotations and/or transformations like `flipped` are not being applied to all icons in the group
## Steps to reproduce

1.
```
<i class="icons small">
  <i data-search-terms="flow, list, timeline" class="clockwise rotated stream icon"></i>
  <i data-search-terms="flow, list, timeline" class="counter clockwise stream icon"></i>
</i>
```

2.
3.

## Expected result
rotations 

## Actual result
no rotations , or partial rotatiion

## Testcase
https://jsfiddle.net/La6sfev8/
## Version

2.x.x

**Comment by ko2in** [comment]
I'm not sure where do you find that the grouped icons? I don't think you can contains `i` tags inside the other `i` tag as group container.

**Comment by ko2in** [comment]
Sorry, my bad. I've looked in the source and indeed there's grouped icon with `icons` variant.

The first child of `icons` has `transform: none` which overrides the transform from `rotated` variant.
```
 i.icons .icon:first-child {
    position: static;
    width: auto;
    height: auto;
    vertical-align: top;
    transform: none;
  }
```

@lubber-de Do you have any idea why the first-child of `icons` need to override the transform property?

**Comment by lubber-de** [comment]
@ko2in By default every icon inside icons group is centered using transform: `translateX(-50%) translateY(-50%)`
When the first icon inside an icons groups does not adjust this, it will be displayed wrongly...i found out nearly all transforms do not really work inside icon groups 🙄 I'll prepare a PR to fix this 🙂 
Just removing the transform: non would make it even worse. See:
![iconstransform](https://user-images.githubusercontent.com/18379884/97895420-0b4c8080-1d34-11eb-8545-270293931d0c.gif)

**Comment by ko2in** [comment]
@lubber-de I see. What's the usage of icons group? The documentation doesn't have icons group example and explanation. I've never used the `icons` before.

**Comment by lubber-de** [comment]
@ko2in uhm... 🤔 icon groups were documented all the time here https://fomantic-ui.com/elements/icon.html#icons
Their purpose is to show multiple icons over each other to get combinations also in different sizes (thats the reaons they are centered)

**Comment by ko2in** [comment]
Ouch. How could I miss that. I've been looking many times the icon page and never noticed the definition tab because I always have been there to look the alias of the icon I want to use :laughing:

**Comment by lubber-de** [comment]
Fixed by #1808
